apps/mobile/src/features/send/send-form/hooks/use-send-form-btc.tsx

```diff
@@ -1,24 +1,77 @@
+import { useAccountUtxosByPaymentType } from '@/queries/balance/bitcoin-balance.query';
+import { useBitcoinAccounts } from '@/store/keychains/bitcoin/bitcoin-keychains.read';
+import { useSettings } from '@/store/settings/settings';
+import { bytesToHex } from '@noble/hashes/utils';
+import * as btc from '@scure/btc-signer';
+
+import { getBtcSignerLibNetworkConfigByMode, payerToBip32Derivation } from '@leather.io/bitcoin';
+import { useAverageBitcoinFeeRates } from '@leather.io/query';
+
 import {
   CreateCurrentSendRoute,
   useSendSheetNavigation,
   useSendSheetRoute,
 } from '../../send-form.utils';
-import { SendFormStxSchema } from '../schemas/send-form-stx.schema';
+import { SendFormBtcSchema } from '../schemas/send-form-btc.schema';
 
 type CurrentRoute = CreateCurrentSendRoute<'send-form-btc'>;
 
+const BTC_TO_SATS = 100000000;
```

Comment:
```suggestion
const SATS_IN_BTC = 100_000_000
```
Can we put this in the `constants` package?